documentation/CP-plugins-documentation/citing.md

```diff
@@ -5,10 +5,10 @@ If using CellProfiler Plugins in a pipeline used in a publication, please cite o
 ENTER CITATION
 
 Additionally, some plugins require citation themselves if used.
-You can determine which plugins require citation by reading the Help information for each plugin, accessible through CellProfiler, or by using the Citation generator built into CellProfiler (since VERSION).
+You can determine which plugins require citation by reading the Help information for each plugin, accessible through CellProfiler, or by using the Citation generator built into CellProfiler (since CellProfiler 5).
 
-To use CellProfiler's citation generator, load your pipeline into CellProfiler.
-Navigate to File => Export => Citation.
+![alt text](images/citation.png)
+Figure 1: How to use the citation tool. To use CellProfiler's citation generator, load your pipeline into CellProfiler (A). Navigate to File => Export => Citation (B). If the module is disabled, it still generates the citation but with the disabled tag. 
```

Comment:
```suggestion
Figure 1: How to use the citation tool. To use CellProfiler's citation generator, load your pipeline into CellProfiler (A). Navigate to File => Export => Citation (B). If any module in your pipeline is currently disabled, a citation for that module will still be generated but it will include a `disabled` tag. 
```